platform/darwin/src/MGLOfflineRegion.h

```diff
@@ -10,6 +10,12 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @protocol MGLOfflineRegion <NSObject>
 
+/**
+ The number of tiles that the object conforming to the `MGLOfflineRegion`
+ protocol contains.
```

Comment:
More precisely, it’s the number of tiles needed to load one of the style’s sources within the region.